code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -33,7 +33,7 @@
 	icon_state = "trashbag"
 	item_state = "trashbag"
 
-	w_class = 4
+	w_class = 3

```

Comment:
This is already a toolbelt+

Don't think it should be able to fit in your backpack too.